docs/api-reference/functions/addEventToActiveSpan.md

```diff
@@ -4,7 +4,7 @@
 function addEventToActiveSpan(event: string, attributes?: Attributes): void;
 ```
 
-Defined in: [api/global.ts:68](https://github.com/adobe/aio-lib-telemetry/blob/ff54ba0c9f0266286f4859c4aab049b808a70c73/source/api/global.ts#L68)
+Defined in: [api/global.ts:72](https://github.com/adobe/aio-lib-telemetry/blob/dd348342643b2b66d5a8c5267221de639b83642e/source/api/global.ts#L72)
```

Comment:
This PR has lots of changes in documentation where 
- the file link is changed
- the line number is changed

Is it correct to imply that for PRs where source is code will cause documentation being regenerated to accommodate links and or line numbers?

I find that it adds noise to PRs as changes not related to the API definition may lead to documentation regeneration.